go.mod

```diff
@@ -1,45 +1,46 @@
 module github.com/antham/chyle
 
 require (
-	github.com/Masterminds/semver v1.3.1
+	github.com/Masterminds/semver v1.3.1 // indirect
 	github.com/Masterminds/sprig v0.0.0-20180403013413-6b2a58267f6a
-	github.com/Sirupsen/logrus v0.11.5
-	github.com/alcortesm/tgz v0.0.0-20161220082320-9c5fe88206d7
-	github.com/anmitsu/go-shlex v0.0.0-20161002113705-648efa622239
+	github.com/Sirupsen/logrus v0.11.5 // indirect
```

Comment:
This looks like a problem. IIUC it means that you have some transitive dependency which is still using the obsolete name.